models/role_permission.go

```diff
@@ -37,6 +37,9 @@ var ROLES_PERMISSIOMS = map[Role][]Permission{
 		SCENARIO_READ, SCENARIO_CREATE, DECISION_READ, DECISION_CREATE, INGESTION,
 		DATA_MODEL_READ, CUSTOM_LISTS_READ, CUSTOM_LISTS_CREATE,
 	},
+	TRANSFER_CHECK_API_CLIENT: {
```

Comment:
Cf the other comment: I want distinct (& more restricted) permissions for the transfer check api than for regular decisions